.travis.yml

```diff
@@ -18,6 +20,12 @@ script:
 # Dump trial log on failure.
 after_failure: "cat _trial_temp/test.log"
 
+# Only run test when committing to master branch.
+branches:
+  only:
+    - master
+    - \d+\.\d+\.\d+
```

Comment:
`branches:` actually applies to tags as well, as far as I can tell, so if we only match master here, version tags will never be built, thus rendering the automatic PyPI releases non-functional.

I wasn't able to actually test this, but I dug through a bunch of Travis documentation and issues to try to figure out what the situation was, and this was the best I could come up with.